addons/sale_timesheet/views/project_update_templates.xml

```diff
@@ -23,19 +42,10 @@ The cost of the project is now at <t t-esc="profitability['costs']"/>, for a rev
 <b><t t-esc="profitability['margin_formatted']"/></b>
 </font>
 <t t-else="" t-esc="profitability['margin_formatted']"/>
-</span> margin (<t t-esc="profitability['margin_percentage']"/> %), and a <b><t t-esc="profitability['billing_rate']"/> % billing rate</b>.
+</span> margin (<t t-esc="profitability['margin_percentage']"/>%).
 </t>
 </div>
         </xpath>
     </template>
 
-    <template id="project_profitability_timesheet_panel" name="Timesheet Profitability">
```

Comment:
Migration script ?

Reply:
This was already done here: https://github.com/odoo/upgrade/pull/2706/files#diff-783ffa09d66a39ead96de3cffcb1b87dac97794c206fc790ee255aaadc56e36eR7